src/pages/userProfile.js

```diff
@@ -70,6 +69,8 @@ const handleCloseModal = (game) => {
                   />
                 </Heading>
               </div>
+              {user[0].profile > 0 && 
```

Comment:
What is line 72 doing? Is this like checking for the length of an array? Checking that the user has a profile?

Reply:
Yep this is to avoid react tryna render anything before it's ready